runtime/near-vm-logic/src/config.rs

```diff
@@ -161,6 +161,18 @@ pub struct ExtCostsConfig {
     /// Base cost for calling a host function.
     pub base: Gas,
 
+    /// Base cost to load in a contract
+    pub contract_load_base: Gas,
+
+    /// Cost to load in contract bytes
+    pub contract_load_byte: Gas,
+
+    /// Base cost of compiling a contract
+    pub contract_compile_base: Gas,
+
+    /// Cost of the execution to compile contract
+    pub contract_compile: Gas,
```

Comment:
Is this cost per byte?

Reply:
No just contract_load_bytes, then it becomes a factor the cost (as yet not provided) of execution of the compiler implementation.

Reply:
Now it is both, should it be renamed with byte at the end?

Reply:
Yes, like others.